examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,882 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `ConditionalDimension` tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial explains how to create equations that only get executed under given conditions.\n",
+    "\n",
+    "As in all our tutorials, we begin by specifying some parameters.\n",
```

Comment:
this sentence sounds odd ... "we begin by specifying some parameters" -- @rhodrin 

I'd just delete it

Reply:
Deleted